README.md

```diff
@@ -24,6 +24,7 @@ The following cryptocurrency exchanges are supported:
 | BitMEX           | x         | x           |   R   O   |
 | Bitstamp         | x         | x           |   R       |
 | Bittrex          | x         | x           | T R       |
+| BL3P             | x         |             | T         |
```

Comment:
The T in the websocket column is for the actual websocket stream. To implement that, you'd have to override OnGetTickersWebSocketAsync().

Reply:
Fixed in 0032baf